### DISCUSSION TITLE
Composer detected issues in your platform

### DISCUSSION THREAD

**Discussion by iknowl97** [DirectoryLister/DirectoryLister#906] [Help & Support] [answered]
How can I solve it ? or skip platform check ?

Fatal error: Composer detected issues in your platform: Your Composer dependencies require the following PHP extensions to be installed: fileinfo. in /www/wwwroot/a.fina.ge/app/vendor/composer/platform_check.php on line 36

Using PHP 8.0 on Ubuntu, got this error after updating DirectoryLister to latest version.

**Comment by PHLAK** [comment, marked as answer]
You will need to install the PHP [fileinfo](https://www.php.net/manual/en/book.fileinfo.php) extension.
